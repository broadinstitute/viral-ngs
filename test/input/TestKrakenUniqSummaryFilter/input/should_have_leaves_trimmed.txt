# KrakenUniq v0.5.7 DATE:2019-11-21T16:03:15Z DB:/tmp/tmp.ohQ3jTMIQh/krakenuniq DB_SIZE:198130386856 WD:/home/dnanexus
# CL:perl /opt/miniconda/envs/viral-ngs-env/bin/krakenuniq --db /tmp/tmp.ohQ3jTMIQh/krakenuniq --threads 32 --fastq-input --gzip-compressed --preload --report-file E023.l16.dedup.krakenuniq-summary_report.txt --paired --output E023.l16.dedup.krakenuniq-reads.txt.gz /tmp/tmp-metagenomics-krakenuniq-jfekrm6y/tmp0eg_wsp4.1.fastq.gz /tmp/tmp-metagenomics-krakenuniq-jfekrm6y/tmpx686dhd3.2.fastq.gz

%	reads	taxReads	kmers	dup	cov	taxID	rank	taxName
27.81	6112	6112	1193206	1.69	NA	0	no rank	unclassified
72.19	15867	11368	349960	3	NA	1	no rank	root
20.47	4499	0	8059	31.5	NA	10239	superkingdom	  Viruses
20.46	4497	0	7930	32	NA	439488	no rank	    ssRNA viruses
20.46	4497	0	7930	32	NA	35301	no rank	      ssRNA negative-strand viruses
20.46	4497	0	7930	32	NA	11157	order	        Mononegavirales
20.46	4497	0	7930	32	NA	11266	family	          Filoviridae
20.46	4497	0	7930	32	NA	186536	genus	            Ebolavirus
20.46	4497	4497	7916	32	NA	186538	species	              Zaire ebolavirus
0.009098	2	0	110	1.71	NA	35237	no rank	    dsDNA viruses, no RNA stage
0.009098	2	0	100	1.6	NA	28883	order	      Caudovirales
0.009098	2	0	93	1.65	NA	10699	family	        Siphoviridae
0.009098	2	0	11	1.73	NA	1982251	genus	          Pa6virus
0.009098	2	0	8	1.62	NA	1982272	species	            Propionibacterium virus PAS50
0.009098	2	2	8	1.62	NA	504553	no rank	              Propionibacterium phage PAS50
