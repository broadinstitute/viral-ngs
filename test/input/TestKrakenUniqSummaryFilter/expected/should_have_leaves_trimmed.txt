# KrakenUniq v0.5.7 DATE:2019-11-21T16:03:15Z DB:/tmp/tmp.ohQ3jTMIQh/krakenuniq DB_SIZE:198130386856 WD:/home/dnanexus
# CL:perl /opt/miniconda/envs/viral-ngs-env/bin/krakenuniq --db /tmp/tmp.ohQ3jTMIQh/krakenuniq --threads 32 --fastq-input --gzip-compressed --preload --report-file E023.l16.dedup.krakenuniq-summary_report.txt --paired --output E023.l16.dedup.krakenuniq-reads.txt.gz /tmp/tmp-metagenomics-krakenuniq-jfekrm6y/tmp0eg_wsp4.1.fastq.gz /tmp/tmp-metagenomics-krakenuniq-jfekrm6y/tmpx686dhd3.2.fastq.gz

%	reads	taxReads	kmers	dup	cov	taxID	rank	taxName
27.810893206534104	6112	6112	1193206	1.69	NA	0	no rank	unclassified
72.1891067934659	15865	11368	349850	3	NA	1	no rank	root
20.46230149701961	4497	0	7949	31.5	NA	10239	superkingdom	  Viruses
20.46230149701961	4497	0	7930	32	NA	439488	no rank	    ssRNA viruses
20.46230149701961	4497	0	7930	32	NA	35301	no rank	      ssRNA negative-strand viruses
20.46230149701961	4497	0	7930	32	NA	11157	order	        Mononegavirales
20.46230149701961	4497	0	7930	32	NA	11266	family	          Filoviridae
20.46230149701961	4497	0	7930	32	NA	186536	genus	            Ebolavirus
20.46230149701961	4497	4497	7916	32	NA	186538	species	              Zaire ebolavirus
